packages/editor/src/layouts/workspace/Stage.vue

```diff
@@ -131,6 +132,10 @@ export default defineComponent({
         services?.editorService.highlight(el.id);
       });
 
+      stage?.on('multiSelect', (els: HTMLElement[]) => {
+        services?.editorService.multiSelect(els);
```

Comment:
multiSelect的参数改成string[]
```suggestion
        services?.editorService.multiSelect(els.map(el => el.id));
```